spec/features/graph/legend_spec.rb

```diff
@@ -15,8 +15,18 @@
         expect(page).to have_content "prometheus_targetpool_duration_ms"
       end
     end
+
+    it "maintains legend state after update" do
+      expect(all('.rickshaw_legend .line.disabled').count).to eq 0
+      legend_elements = all('.rickshaw_legend .action')
+      legend_elements.first.click
+      expect(all('.rickshaw_legend .line.disabled').count).to eq 1
+      open_tab 'Refresh'
```

Comment:
How does open_tab work. Does it make sure that all elements have been updated already? Can we ensure that something which _should have_ updated got updated before the line.disabled check?